.eslintrc

```diff
@@ -9,6 +9,12 @@
     "unicorn/prefer-module": "off",
 
     // TODO: This can be removed if we decide to add jsdoc to all functions
-    "valid-jsdoc": "off"
+    "valid-jsdoc": "off",
+    "no-await-in-loop": "off",
+    "unicorn/filename-case": "off",
```

Comment:
we use kebab case in this repo so this can be removed and `baseCommand` can be updated to `base-command`